packages/functional-tests/pages/login.ts

```diff
@@ -219,12 +188,44 @@ export class LoginPage extends BaseLayout {
     return this.page.fill(selectors.PASSWORD, password);
   }
 
+  async showPasswordMouseAction() {
+    return Promise.all([
+      await this.page.locator(selectors.PASSWORD).type('password'),
+      await this.page.locator(selectors.SHOW_PASSWORD).hover(),
+      await this.page.mouse.down(),
```

Comment:
I don't understand why we'd need a Promise.all here.  But if we do use it, I don't think we need to `await` the promises in the list.

Reply:
Keeping `Promise.all` and removing `await` would not perform the rest of the steps in the `function()` block and is giving me "unable to find selector" error. But yes I can remove the `Promise.all `entirely.